setup.py

```diff
@@ -46,9 +45,6 @@
         'Natural Language :: English',
         'License :: OSI Approved :: BSD License',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6',
```

Comment:
I’m not sure about dropping Python support. I think it may be safer to do this _after_ the rebranding.

Reply:
We must at least drop 2.7 to address outdated dependencies. 

If we drop 2.7, we should take the chance to drop 3.5 and 3.6, since no projects should be using them.

If there are projects with 2.7, 3.5, or 3.6, they should be able to use the current version of `shub` as is.

Reply:
Given the drop of 3.6, we should hold this pull request until Dec 23.